master/buildbot/status/web/builder.py

```diff
@@ -647,3 +647,11 @@ def getChild(self, path, req):
 
         return HtmlResource.getChild(self, path, req)
 
+    def getBranchName(self):
+        branch_name = ''
+        if len(self.project.codebases) > 0:
+            branch = self.project.codebases[0].itervalues().next()['branch']
+            if len(branch) > 0:
```

Comment:
not checking for KeyError (missing 'branch'), because if it's missing Katana breaks somewhere else anyway